frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -1,35 +1,37 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useActions, useValues } from 'kea'
 import dayjs from 'dayjs'
 import { parsePeopleParams, trendsLogic } from 'scenes/trends/trendsLogic'
 import { DownloadOutlined } from '@ant-design/icons'
-import { Modal, Button, Spin } from 'antd'
+import { Modal, Button, Spin, Input } from 'antd'
 import { PersonsTable } from 'scenes/persons/PersonsTable'
-import { Link } from 'lib/components/Link'
-import { ArrowRightOutlined, ClockCircleOutlined } from '@ant-design/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { ViewType } from 'scenes/insights/insightLogic'
-
+import { FEATURE_FLAGS } from 'lib/constants'
 interface Props {
     visible: boolean
     view: ViewType
     onSaveCohort: () => void
 }
 
 export function PersonModal({ visible, view, onSaveCohort }: Props): JSX.Element {
-    const { people, filters, peopleModalURL, loadingMorePeople } = useValues(
+    const { people, filters, loadingMorePeople, firstLoadedPeople } = useValues(
         trendsLogic({ dashboardItemId: null, view })
     )
-    const { setShowingPeople, loadMorePeople } = useActions(trendsLogic({ dashboardItemId: null, view }))
+    const { setPersonsModalFilters } = useActions(trendsLogic({ dashboardItemId: null, view }))
+    const { setShowingPeople, loadMorePeople, setPeople } = useActions(trendsLogic({ dashboardItemId: null, view }))
     const { featureFlags } = useValues(featureFlagLogic)
-
+    const [searchTerm, setSearchTerm] = useState('')
```

Comment:
Nope :). Let's keep data in the data layer. If you already have "personsModalFilters" in the trends logic, then why mix concerns and keep the state of one of those filters tied to a React component?

Reply:
I wanted to avoid adding more things to `trendsLogic` since that file is becoming huge, but maybe I should refactor the people stuff and this searchTerm state out into its own personsModalLogic instead?

Reply:
I think I'll do that, unless there's some side effects of doing this that I'm not currently aware of

Reply:
I think this makes a lot of sense! :)

Reply:
Hmm I seem to be having a hard time connecting the trendsLogic since it requires props to the new personsModalLogic, I won't refactor it for now and I've removed the searchTerm from using `useState`. Is there something special I have to do to connect a logic with props to another logic?

Reply:
I guess it's fine to keep as is then. Connecting these together with props will probably get rather complicated :/.

In any case, to connect logic with props, you should just pass in the props as normally, so `trendsLogic({...}).selectors.bla` or whatever you need to do. These props you can probably get from the props to the personLogic (e.g. `props: { trendsLogicProps: { ... } }`, but at the end of the day, this currently makes things feel complicated. 

However the only problem now is that when you close and reopen the modal, the searchfield value is filled in. I'm not sure if this is the expected behaviour.